# Profitable App to Launch based on Data Analysis

Our aim in this project is to identify competitive mobile device profiles for both the App Store and Google Play markets. Our job is to make data-driven decisions about the kind of applications that a team of developers need to create.

We are focussing on free apps, which generate revenue from in-app ads. This means that our income for any particular app depends mainly on the number of users using the application. Our mission is to evaluate data and help our developers recognize what sort of apps more consumers are likely to use. 

## Opening and Exploring Data

There are approximately 2 million iOS apps available on the App Store, and 2.1 million Android apps on Google Play.

Collecting data for over four million apps requires a significant amount of time and money, so we'll try to analyze a sample of data instead. To avoid spending resources with collecting new data ourselves, we should first try to see whether we can find any relevant existing data at no cost. Luckily, these are two data sets that seem suitable for our purpose:

* A data set containing data about approximately ten thousand Android apps from Google Play. You can download the data set directly from this [link](https://dq-content.s3.amazonaws.com/350/googleplaystore.csv).
* A data set containing data about approximately seven thousand iOS apps from the App Store. You can download the data set directly from this [link](https://dq-content.s3.amazonaws.com/350/AppleStore.csv).

Let's start by opening the two datasets and then continue exploring the data.

In [1]:
from csv import reader

#Creating a function which converts csv files to datasets of lists.
def csv_to_list(a_file, head = True):
    opened_file = open(a_file, encoding = 'utf-8')
    read_file = reader(opened_file)
    dataset = list(read_file)
    header = dataset[0]
    dataset = dataset[1:]
    return dataset, header

#Importing Datasets and converting them
gplay, gplay_header = csv_to_list('googleplaystore.csv')
appstore, appstore_header = csv_to_list('AppleStore.csv')

To make it easier to explore the two data sets, we'll first write a function named explore_data() that we can use repeatedly to explore rows in a more readable way. We'll also add an option for our function to show the number of rows and columns for any data set.

In [2]:
def explore_data(dataset, start, end, rows_and_column = False):
    sliced_data = dataset[start:end]
    for x in sliced_data:
        print(x)
        print('\n')
    if rows_and_column:
        print('Number of rows are - ', len(dataset))
        print('Number of columns are - ', len(dataset[0]))
        
print(gplay_header)
print('\n')
explore_data(gplay, 0, 3, True)

['App', 'Category', 'Rating', 'Reviews', 'Size', 'Installs', 'Type', 'Price', 'Content Rating', 'Genres', 'Last Updated', 'Current Ver', 'Android Ver']


['Photo Editor & Candy Camera & Grid & ScrapBook', 'ART_AND_DESIGN', '4.1', '159', '19M', '10,000+', 'Free', '0', 'Everyone', 'Art & Design', 'January 7, 2018', '1.0.0', '4.0.3 and up']


['Coloring book moana', 'ART_AND_DESIGN', '3.9', '967', '14M', '500,000+', 'Free', '0', 'Everyone', 'Art & Design;Pretend Play', 'January 15, 2018', '2.0.0', '4.0.3 and up']


['U Launcher Lite – FREE Live Cool Themes, Hide Apps', 'ART_AND_DESIGN', '4.7', '87510', '8.7M', '5,000,000+', 'Free', '0', 'Everyone', 'Art & Design', 'August 1, 2018', '1.2.4', '4.0.3 and up']


Number of rows are -  10841
Number of columns are -  13


We see that the Google Play data set has 10841 apps and 13 columns. At a quick glance, the columns that might be useful for the purpose of our analysis are 'App', 'Category', 'Reviews', 'Installs', 'Type', 'Price', and 'Genres'.

Now let's take a look at the App Store data set.

In [3]:
print(appstore_header)
print('\n')
explore_data(appstore, 0, 3, True)

['id', 'track_name', 'size_bytes', 'currency', 'price', 'rating_count_tot', 'rating_count_ver', 'user_rating', 'user_rating_ver', 'ver', 'cont_rating', 'prime_genre', 'sup_devices.num', 'ipadSc_urls.num', 'lang.num', 'vpp_lic']


['284882215', 'Facebook', '389879808', 'USD', '0.0', '2974676', '212', '3.5', '3.5', '95.0', '4+', 'Social Networking', '37', '1', '29', '1']


['389801252', 'Instagram', '113954816', 'USD', '0.0', '2161558', '1289', '4.5', '4.0', '10.23', '12+', 'Photo & Video', '37', '0', '29', '1']


['529479190', 'Clash of Clans', '116476928', 'USD', '0.0', '2130805', '579', '4.5', '4.5', '9.24.12', '9+', 'Games', '38', '5', '18', '1']


Number of rows are -  7197
Number of columns are -  16


We have 7197 iOS apps in this data set, and the columns that seem interesting are: 'track_name', 'currency', 'price', 'rating_count_tot', 'rating_count_ver', and 'prime_genre'.

## Cleaning Data

In [4]:
for x in appstore:
    if x[5] == 'USD':
        print(x)

In [5]:
del gplay[10472]

In [6]:
def delete_row(a_list, term, column):
    temp_list = []
    for x in a_list:
        if x[column] != term:
            temp_list.append(x)
    return temp_list

#### Checking Duplicate Apps

In [7]:
duplicate_apps = []
unique_apps = []

for app in gplay:
    name = app[0]
    if name in unique_apps:
        duplicate_apps.append(name)
    else:
        unique_apps.append(name)
    
print('Number of duplicate apps:', len(duplicate_apps))
print('\n')
print('Examples of duplicate apps:', duplicate_apps[:15])

Number of duplicate apps: 1181


Examples of duplicate apps: ['Quick PDF Scanner + OCR FREE', 'Box', 'Google My Business', 'ZOOM Cloud Meetings', 'join.me - Simple Meetings', 'Box', 'Zenefits', 'Google Ads', 'Google My Business', 'Slack', 'FreshBooks Classic', 'Insightly CRM', 'QuickBooks Accounting: Invoicing & Expenses', 'HipChat - Chat Built for Teams', 'Xero Accounting Software']


In [8]:
#Removing Multiple Entries
def remove_multiple(a_list, index):
   
    review_count = {}
    for x in a_list:
        name = x[0]
        if name not in review_count:
            review_count[name] = float(x[index])
        elif float(x[index]) > review_count[name]:
            review_count[name] = float(x[index])
    clean_list = []
    already_added = []
    for x in a_list:
        name = x[0]
        if (float(x[index]) == review_count[name]) and (name not in already_added):
            clean_list.append(x)
            already_added.append(name)
    
    return clean_list


            
gplay = remove_multiple(gplay, 3)
appstore = remove_multiple(appstore, 5)

#### Filtering for English only Apps

English language's ASCII codes range from 0-127, if the character is out range, then it's most probably another language.

If we remove all the apps with single character exceeding the range, we might even skip the english apps which have emojis. Therefore, we will remove the apps which have more than 3 characters who are out of range.

In [9]:
def is_english(a_list, index):
    temp_list = []
    for x in a_list:
        name = x[index]
        counter = 0
        for y in name:
            if ord(y) > 127:
                counter += 1
        if counter < 3:
            temp_list.append(x)
        counter = 0
    return temp_list

gplay = is_english(gplay, 0)
appstore = is_english(appstore, 1)

print(len(gplay))
print(len(appstore))

9597
6155


#### Isolating Free Apps

In [10]:
def free_apps(a_list, index):
    temp_list = []
    for x in a_list:
        price = x[index]
        if price == '0':
            temp_list.append(x)
        elif price == '0.0':
            temp_list.append(x)
    return temp_list    

gplay = free_apps(gplay, 7)
appstore = free_apps(appstore, 4)

print(len(gplay))
print(len(appstore))

8848
3203


___

## Analysing Data

As we stated in the introduction, our goal is to determine the types of apps that are likely to draw more customers, as our revenue is strongly affected by the amount of people using our applications.

Our validation approach for an product concept is comprised of three measures to reduce costs and overhead:
* Create and add a limited edition of the Android app to Google Play
* If the app has a strong consumer reaction, we'll further improve it
* If the product is successful after six months, we'll create and attach an iOS edition of the app to the App Store

Since our ultimate goal is to introduce the app to both Google Play and the App Store, we need to identify good user profiles in both markets. A profile that fits well for both sectors , for example, may be a productivity app that makes use of gamification.

Let's continue the analysis by having a sense of what the most popular genres are for any market. For this we would need to build frequency tables in our data sets for a few columns.

### Finding the most common genre

In [11]:
#Making a fucntion which takes in a list and index and returns the percentages of the column.

def column_percentage(a_list, index):
    column_dict = {}
    for x in a_list:
        if x[index] in column_dict:
            column_dict[x[index]] += 1
        else:
            column_dict[x[index]] = 1
    
    #finding total entries
    total = 0
    for x in column_dict:
        total += column_dict[x]
    
    #Updating dictionary to percentage
    for x in column_dict:
        column_dict[x] = float((column_dict[x]/total)*100)
    
    return sort_dict(column_dict)
    
def sort_dict(a_dict): 
    table_display = []
    for x in a_dict:
        key_val = (a_dict[x], x)
        table_display.append(key_val)
    table_sorted = sorted(table_display, reverse = True)
    list_sorted = list(table_sorted)
    return list_sorted

**Appstore Most Common Genres**

Appstore dataset has the column 'prime_genre' and we analysed that below.

In [12]:
genre_percent_appstore = column_percentage(appstore, -5)
explore_data(genre_percent_appstore, 0, 5)

(58.25788323446769, 'Games')


(7.836403371838902, 'Entertainment')


(4.995316890415236, 'Photo & Video')


(3.6840462066812365, 'Education')


(3.3093974399000934, 'Social Networking')




We can see that more than a half (58.16 per cent) of the free English applications are games. Entertainment applications are approximately 8%, led by photo and video apps, close to 5%. Just 3.66% of the applications are built for education, accompanied by social networking apps that make up 3.29% of the apps in our data set.

The general impression is that the App Store (at least the section that includes free English content) is overrun by applications built for fun (games, entertainment, photo and video, social networking, sports , music, etc.) whereas practical applications (education, shopping, services, efficiency, lifestyle, etc.) are more uncommon. Although the fact that fun applications are the most popular still doesn't mean that they will have the highest amount of users — the demand may not be the same as the supply.

**Google Play Store Most Common Genres**

Google Play dataset has two similar columns, "Category" and "Genres". We will analyse both the datasets below.

In [13]:
# Analysing the column "Category"

genre_percent_gplay = column_percentage(gplay, 1)
explore_data(genre_percent_gplay, 0, 5)

(18.942133815551536, 'FAMILY')


(9.697106690777577, 'GAME')


(8.453887884267631, 'TOOLS')


(4.599909584086799, 'BUSINESS')


(3.899186256781193, 'PRODUCTIVITY')




In [14]:
# Analysing the column "Genres"

genre_percent_gplay = column_percentage(gplay, 9)
explore_data(genre_percent_gplay, 0, 5)

(8.44258589511754, 'Tools')


(6.080470162748644, 'Entertainment')


(5.357142857142857, 'Education')


(4.599909584086799, 'Business')


(3.899186256781193, 'Productivity')




On Google Play, the environment appears dramatically different: there aren't too many applications built for entertainment, and a large number of applications seem to be developed for practical purposes (family, resources, company, lifestyle, efficiency, etc.). However, if we further examine this, we will see that the family group (which accounts for approximately 19 percent of the apps) mainly includes games for children.

Even so, it seems that realistic applications are best portrayed on Google Play relative to the App Store. The frequency table we see for the Genres column also supports this picture

The distinction between the columns of the Genres and the columns of the Category is not perfectly clear but one aspect we should note is that the column of Genres is far more granular (it has more categories). At the moment we 're just aiming at the larger picture and we're only going to deal by moving on in the Category column.

Up until this point , we found that applications made for fun rule the App Store, while Google Play presents a more diverse ecosystem of both realistic and non-fun devices. Now we would like to get an understanding about what kind of apps most users have.

---

### Finding the most popular genre on Appstore

One way to figure out which genres are most popular (have the most users) is by measuring the total amount of installs for each genre of apps. For the Google Play data set, we can find this information in the Installs column,  but this detail is lacking for the App Store data set.  As a workaround, we'll take the total number of user ratings as a proxy, which we can find in the rating_count_tot app.

#### Finding the average number of user ratings per app genre on the App Store

To do that, we'll need to:
* Isolate the apps of each genre
* Sum up the user ratings for the apps of that genre
* Divide the sum by the number of apps belonging to that genre

In [15]:
#Function to calculate the above

def avg_by_ratings(a_list, genre_index, rating_count_index):
    column_dict = {}
    for x in a_list:
        if x[genre_index] in column_dict:
            column_dict[x[genre_index]] += 1
        else:
            column_dict[x[genre_index]] = 1
    
    ratings_sum_dict = {}
    for x in a_list:
        if x[genre_index] in ratings_sum_dict:
            ratings_sum_dict[x[genre_index]] += float(x[rating_count_index])
        else:
            ratings_sum_dict[x[genre_index]] = float(x[rating_count_index])
    
    #finding total entries
    total = 0
    for x in column_dict:
        total += column_dict[x]        
    
    percentage_dict = {}
    for x in column_dict:
        percentage_dict[x] = (ratings_sum_dict[x]/column_dict[x])*100
    
    return sort_dict(percentage_dict)
    

In [16]:
popular_appstore = avg_by_ratings(appstore, -5, 5)
explore_data(popular_appstore, 0, 5)

(8609033.333333332, 'Navigation')


(7935047.05882353, 'Reference')


(7154834.905660378, 'Social Networking')


(5732653.03030303, 'Music')


(5227989.285714285, 'Weather')




Navigation applications have the largest amount of user ratings on average, although this statistic is highly skewed by Waze and Google Maps, who have almost half a million user reviews combined

In [17]:
for x in appstore:
    if x[-5] == "Navigation":
        print(x[1], ':', x[5])

Waze - GPS Navigation, Maps & Real-time Traffic : 345046
Google Maps - Navigation & Transit : 154911
Geocaching® : 12811
CoPilot GPS – Car Navigation & Offline Maps : 3582
ImmobilienScout24: Real Estate Search in Germany : 187
Railway Route Search : 5


The same trend extends to social networking apps, where the total amount is highly skewed by a few giants such as Twitter , Instagram, Skype etc. The same happens to music applications, where the total figure is highly skewed by a few major players like Itunes , Spotify and Shazam.

Reference applications have an overall consumer ranking of 74,942, but in reality it is the Bible and Dictionary.com that have inflated the total rating:

In [18]:
for x in appstore:
    if x[-5] == "Reference":
        print(x[1], ':', x[5])

Bible : 985920
Dictionary.com Dictionary & Thesaurus : 200047
Dictionary.com Dictionary & Thesaurus for iPad : 54175
Google Translate : 26786
Muslim Pro: Ramadan 2017 Prayer Times, Azan, Quran : 18418
New Furniture Mods - Pocket Wiki & Game Tools for Minecraft PC Edition : 17588
Merriam-Webster Dictionary : 16849
Night Sky : 12122
City Maps for Minecraft PE - The Best Maps for Minecraft Pocket Edition (MCPE) : 8535
LUCKY BLOCK MOD ™ for Minecraft PC Edition - The Best Pocket Wiki & Mods Installer Tools : 4693
GUNS MODS for Minecraft PC Edition - Mods Tools : 1497
Guides for Pokémon GO - Pokemon GO News and Cheats : 826
WWDC : 762
Horror Maps for Minecraft PE - Download The Scariest Maps for Minecraft Pocket Edition (MCPE) Free : 718
VPN Express : 14
Real Bike Traffic Rider Virtual Reality Glasses : 8
Jishokun-Japanese English Dictionary & Translator : 0


The niche seems to be showing some promise, however;  one thing we can do is take another famous book and convert it into an app where besides the raw book edition, we should incorporate various apps. These may include regular book quotations, an audible recording of the book, book quizzes etc. We might even integrate a dictionary into the app, meaning people don't have to quit our app to look up terms in an unrelated site.

The concept seems to suit well with the reality that for-fun applications rule the App Store. It means that the competition might be a little crowded with for-fun apps, suggesting that an useful app would have the potential to stick out from the large amount of applications on the App Store.

Certain categories that tend to be common include weather, book, food and drink, or finance. The genre of the book seems to correlate a bit with the theme of the app that we mentioned above, but the other genres do not seem that important to us:
* Weather applications — users usually don't spend that much time in the app, so there's no chance of profiting from in-app adds. It could also require us to link our apps to non-free APIs to get accurate live weather data.
* Food and drink - Starbucks, Dunkin' Donuts, McDonald's, etc. are examples here. So creating a famous food and drink app needs true cooking and distribution operation, which is outside our company's reach.
* Apps for finance — such features provide insurance, bill paying, money transfer, etc. Creating a finance app calls for awareness of the subject and we don't want to employ a finance specialist just to create an app.

Now let's look a little more at the demand for Google Play.

### Finding the most popular genre on Google Play

We already have details on the amount of downloads for the Google Play market and we will be able to get a better view of the success of the genre. Nonetheless, the number of installations does not seem specific enough — we can see that the bulk of numbers are open-ended (100 +, 1,000 +, 5,000 +, etc.)

In [19]:
def find_unique(a_list, index):
    installs_dict = {}
    for x in a_list:
        if x[index] in installs_dict:
            installs_dict[x[index]] += 1
        else:
            installs_dict[x[index]] = 1
            
    return sort_dict(installs_dict)

gplay_installs = find_unique(gplay, 5)
explore_data(gplay_installs, 0, 5)

(1394, '1,000,000+')


(1021, '100,000+')


(935, '10,000,000+')


(902, '10,000+')


(743, '1,000+')




The issue with the data is that it isn't precise. For example, we don't know whether an app that has 100,000 + installs has 100,000, 200,000, or 350,000 installs. Nonetheless, for our purposes, we don't require really accurate numbers — we only want to get an understanding of which device categories are drawing the most consumers, so we don't need absolute accuracy in terms of number of consumers.

We're going to leave the numbers as they are, which means that we'll consider that an app with 100,000+ installs has 100,000 installs, and an app with 1,000,000+ installs has 1,000,000 installs, and so on.



To perform computations, however, we'll need to convert each install number to float — this means that we need to remove the commas and the plus characters, otherwise the conversion will fail and raise an error. We'll do this directly in the loop below, where we also compute the average number of installs for each genre (category).

In [21]:
# Function to remove commas and plus, and find avg
def make_number(a_list, index, index_c):
    temp1_list = []
    for x in a_list:
        temp1_list.append(x)
    temp_list = []
    for x in temp1_list:
        installs = str(x[index])
        installs = installs.replace(',','')
        installs = installs.replace('+','')
        installs = installs.replace(' ','')
        x[index] = float(installs)
        temp_list.append(x)
    
    freq_installs = {}
    genre_count = {}
    total_installs = 0
    for x in temp_list:
        if x[index_c] in freq_installs:
            freq_installs[x[index_c]] += x[index]
            genre_count[x[index_c]] += 1
        else:
            freq_installs[x[index_c]] = x[index]
            genre_count[x[index_c]] = 1
        total_installs += x[index]

    avg_installs = {}
    for x in freq_installs:
        avg_installs[x] = freq_installs[x] / genre_count[x]
    
    return sort_dict(avg_installs)

gplay_avg = make_number(gplay, 5, 1)
explore_data(gplay_avg, 0, 10)

(38590581.08741259, 'COMMUNICATION')


(24727872.452830188, 'VIDEO_PLAYERS')


(23253652.127118643, 'SOCIAL')


(17840110.40229885, 'PHOTOGRAPHY')


(16787331.344927534, 'PRODUCTIVITY')


(15544014.51048951, 'GAME')


(13984077.710144928, 'TRAVEL_AND_LOCAL')


(11640705.88235294, 'ENTERTAINMENT')


(10830251.970588235, 'TOOLS')


(9549178.467741935, 'NEWS_AND_MAGAZINES')




Communication applications have the most installs, on average: 38,590,581. This figure is severely skewed by a few devices with over one billion installs (WhatsApp, Facebook Messenger, Skype, Google Chrome, Gmail, and Hangouts), and a few others with more than 100 million and 500 million installs:

In [32]:
for x in gplay:
    if x[1] == 'COMMUNICATION' and (x[5] == 1000000000 or x[5] == 500000000 or x[5] == 100000000):
        print(x[0], ':', x[5],'+')

WhatsApp Messenger : 1000000000.0 +
imo beta free calls and text : 100000000.0 +
Android Messages : 100000000.0 +
Google Duo - High Quality Video Calls : 500000000.0 +
Messenger – Text and Video Chat for Free : 1000000000.0 +
imo free video calls and chat : 500000000.0 +
Skype - free IM & video calls : 1000000000.0 +
Who : 100000000.0 +
GO SMS Pro - Messenger, Free Themes, Emoji : 100000000.0 +
LINE: Free Calls & Messages : 500000000.0 +
Google Chrome: Fast & Secure : 1000000000.0 +
Firefox Browser fast & private : 100000000.0 +
UC Browser - Fast Download Private & Secure : 500000000.0 +
Gmail : 1000000000.0 +
Hangouts : 1000000000.0 +
Messenger Lite: Free Calls & Messages : 100000000.0 +
Kik : 100000000.0 +
KakaoTalk: Free Calls & Text : 100000000.0 +
Opera Mini - fast web browser : 100000000.0 +
Opera Browser: Fast and Secure : 100000000.0 +
Telegram : 100000000.0 +
Truecaller: Caller ID, SMS spam blocking & Dialer : 100000000.0 +
UC Browser Mini -Tiny Fast Private & Secure : 1000000

For the category of video players we see the same trend which is the runner-up with 24,727,872 installs. Apps such as the Youtube , Google Play Movies & TV, or MX Player control the business. For social applications (where we have giants like Twitter, Snapchat, Google+, etc.), photography apps (Google Images and other common picture editors), or productivity apps (Microsoft Word, Dropbox, Google Calendar, Evernote, etc.), the trend is replicated. 

The big problem is again that these types of apps may seem more common than they actually are. In fact, these niches tend to be occupied by a few giants that are impossible to compete with.

The game genre seems fairly common, but we've pointed out previously that this section of the market seems a little crowded, so if possible we 'd like to come up with a specific app suggestion. 

The books and genre of comparison also seems relatively common, with an estimated number of 8,767,811 installs. Exploring this in more detail is fascinating, because we find this genre has some potential to perform well on the App Store, and our intention is to suggest an device genre that demonstrates potential to be competitive on both the App Store and Google Play as well.

Let's look at some of the genre's applications and their amount of installations:

In [27]:
for x in gplay:
    if x[1] == 'BOOKS_AND_REFERENCE':
        print(x[0], ':', x[5])

E-Book Read - Read Book for free : 50000.0
Download free book with green book : 100000.0
Wikipedia : 10000000.0
Cool Reader : 10000000.0
Free Panda Radio Music : 100000.0
Book store : 1000000.0
FBReader: Favorite Book Reader : 10000000.0
English Grammar Complete Handbook : 500000.0
Free Books - Spirit Fanfiction and Stories : 1000000.0
Google Play Books : 1000000000.0
AlReader -any text book reader : 5000000.0
Offline English Dictionary : 100000.0
Offline: English to Tagalog Dictionary : 500000.0
FamilySearch Tree : 1000000.0
Cloud of Books : 1000000.0
Recipes of Prophetic Medicine for free : 500000.0
ReadEra – free ebook reader : 1000000.0
Anonymous caller detection : 10000.0
Ebook Reader : 5000000.0
Litnet - E-books : 100000.0
Read books online : 5000000.0
English to Urdu Dictionary : 500000.0
eBoox: book reader fb2 epub zip : 1000000.0
English Persian Dictionary : 500000.0
Flybook : 500000.0
All Maths Formulas : 1000000.0
Ancestry : 5000000.0
HTC Help : 10000000.0
English translatio

The book and reference category covers a range of applications: ebook editing and reading tools, different libraries, dictionaries, computer or language tutorials, etc. There also appears to be a limited range of extremely popular apps that were skewing the average:

In [31]:
for x in gplay:
    if x[1] == 'BOOKS_AND_REFERENCE' and (x[5] == 1000000
                                            or x[5] == 5000000
                                            or x[5] == 10000000
                                            or x[5] == 50000000):
        print(x[0], ':', x[5],'+')

Wikipedia : 10000000.0 +
Cool Reader : 10000000.0 +
Book store : 1000000.0 +
FBReader: Favorite Book Reader : 10000000.0 +
Free Books - Spirit Fanfiction and Stories : 1000000.0 +
AlReader -any text book reader : 5000000.0 +
FamilySearch Tree : 1000000.0 +
Cloud of Books : 1000000.0 +
ReadEra – free ebook reader : 1000000.0 +
Ebook Reader : 5000000.0 +
Read books online : 5000000.0 +
eBoox: book reader fb2 epub zip : 1000000.0 +
All Maths Formulas : 1000000.0 +
Ancestry : 5000000.0 +
HTC Help : 10000000.0 +
Moon+ Reader : 10000000.0 +
English-Myanmar Dictionary : 1000000.0 +
Golden Dictionary (EN-AR) : 1000000.0 +
All Language Translator Free : 1000000.0 +
Aldiko Book Reader : 10000000.0 +
Dictionary - WordWeb : 5000000.0 +
50000 Free eBooks & Free AudioBooks : 5000000.0 +
Al-Quran (Free) : 10000000.0 +
Al Quran Indonesia : 10000000.0 +
Al'Quran Bahasa Indonesia : 10000000.0 +
Al Quran Al karim : 1000000.0 +
Al Quran : EAlim - Translations & MP3 Offline : 5000000.0 +
Koran Read &MP3 30

This market appears to be filled by ebook scanning and reading applications, as well as numerous libraries and dictionaries, and creating related apps definitely isn't a smart idea because there would be some serious rivalry. 

We also note that there are quite a few apps developed around the Quran book and indicate that it could be lucrative to develop an app around a common book. This seems that having a famous book (maybe a more recent novel) and transforming it into an app might be beneficial for both the industries of Google Play and the App Store.

It seems like the market is still full of libraries, though, and we ought to incorporate any extra features apart from the book's original edition. These may include frequent quotations from the novel, an audible edition of the novel, book quizzes, a website where people will chat about the book etc.

## Conclusions

In this project , we analyzed data on smartphone applications from the App Store and Google Play with the goal of suggesting an app profile that could be competitive for both markets. 

We decided that it would be beneficial for both the Google Play and the App Store markets to take a famous book (perhaps a more recent novel) and convert it into an app. The markets are now full of libraries and in addition to the raw edition of the novel, we need to introduce certain unique features. These may include frequent quotations from the novel, an audible edition of the novel, book quizzes, a website where people will chat about the book etc.